packages/open-next/src/adapters/cache.ts

```diff
@@ -76,151 +77,153 @@ interface CacheHandlerValue {
 
 type Extension = "json" | "html" | "rsc" | "body" | "meta" | "fetch";
 
-const FETCH_CACHE_KEY_PREFIX = "__fetch";
+// Expected environment variables
+const { CACHE_BUCKET_NAME, CACHE_BUCKET_KEY_PREFIX } = process.env;
 
 export default class S3Cache {
   private client: S3Client;
   private buildId: string;
+
   constructor(_ctx: CacheHandlerContext) {
-    this.client = new S3Client({ region: process.env.ORIGIN_REGION });
-    this.buildId = loadBuildId(path.dirname(_ctx.serverDistDir ?? ".next/server"));
+    this.client = new S3Client({});
```

Comment:
You removed the region parameter here, i think it will cause an error for lambda@edge since the lambda@edge function might be in a different region than the bucket.